index.html

```diff
@@ -26,10 +26,10 @@
                 </section>
                 <div id="mobile-menu" class="list-mobile">
                     <span class="close">x</span>
-                    <ul>
+                    <ul class="ul-close">
                         <li><a href="/about.html">About</a></li>
-                        <li><a href="#"></a>Biography</li>
-                        <li><a href="#"></a>Projects</li>
+                        <li><a href="#champions">Facts</a></li>
+                        <li><a href="#architect">Speakers</a></li>
```

Comment:
- [ ] Your about file seems to be broken. take a quick look through to make it work, check the screenshot below:
![about1](https://user-images.githubusercontent.com/55635977/128219073-8387fd11-c07b-4740-a98c-3e84c8b8eb8d.png)

Reply:
I don't know what's happen with github pages but if you saw my [video](https://www.loom.com/share/0ea8113063bf40e4995f24516481e93a) you will see that my project works fine!

Reply:
This issue is well resolved! ✔️